.circleci/config.yml

```diff
@@ -1,13 +1,28 @@
 version: 2.1
 orbs:
   azure-cli: circleci/azure-cli@1.1.0
-jobs:
-  install:
+
+executors:
+  default-executor:
+    docker:
+      - image: cimg/node:15.3
+    working_directory: ~/react-spectrum
+
+  large-executor:
     docker:
-      - image: circleci/node:12
+      - image: cimg/node:15.3
```

Comment:
Why are we updating node for react 17 tests? I'm still using 12 locally? Will I also need to upgrade? And what is cimg?

Reply:
Looks like i conflated two issues
one is the node upgrade to fix a failing test, the other is to upgrade react 17, i'll split it out

as for upgrading node, yes, but you should be using nvm, so it's an easy thing to do

Reply:
cimg is their new way of naming circleci images

Reply:
waiting for this to build and make sure everything is good before removing the react 17 upgrade stuff from this branch https://github.com/adobe/react-spectrum/pull/1354